sale_product_set/models/product_set.py

```diff
@@ -1,11 +1,18 @@
 # -*- encoding: utf-8 -*-
-from openerp import fields, models, _
+from openerp import api, fields, models, _
 
 
 class ProductSet(models.Model):
     _name = 'product.set'
     _description = 'Product set'
 
-    name = fields.Char(_('Name'), help=_('Product set name'), required=True)
+    name = fields.Char(u"Name", help=u"Product set name", required=True)
     set_line_ids = fields.One2many(
-        'product.set.line', 'product_set_id', _(u"Products"))
+        'product.set.line', 'product_set_id', u"Products", copy=True)
+
+    @api.one
```

Comment:
I agree with the web interface you cannot copy more than one record, but with the API or xmlrpc or in a module it might be useful to be able to do a mass copy.